### DISCUSSION TITLE
Double-opening of minimised app

### DISCUSSION THREAD

**Discussion by hencomb** [lawand-dot-io/taskbar#66] [Feature requests]
Hey all,

Fantastic app, love it.

I noticed the following odd behaviour: if an open app is hidden with cmd-H and then clicking on the icon in taskbar it re-opens it fine.
However, if an open app is minimised and then re-opened by clicking on the icon in taskbar, it opens twice: ie, it opens, closes, then re-opens - almost like a quick-flash.

**Comment by lawand-dot-io** [comment]
Hello. I'm glad you're enjoying Taskbar. Please explain the steps to reproduce the bug and maybe tell me which app is an example of this behavior and how many windows this app has open.

**Comment by hencomb** [comment]
HI there,

There are six apps open. It basically happens with all apps.

I am using Sanoma.

To reproduce, I installed the taskbar app, and changed a few settings (see below).
Only happens if an app is manually minimised with the yellow icon.

See video here: 

https://github.com/user-attachments/assets/8bd59029-843f-45c2-8c3e-b0724477afed

**Comment by lawand-dot-io** [comment]
Interesting, haven't seen that before. I wonder why you don't have minimize animation. Is it because you checked the "Fully hide Dock" option? The other question is, does the same thing happen when you use the Dock? So, minimize calculator using the yellow button then click on the calculator icon in the Dock

**Comment by hencomb** [comment]
Howzit,

I tried switching off “Fully hide Dock”, same result.

Using the Dock it works as expected, no double-open.

Settings are currently using the Genie effect animation to minimize – this works when using the standard Dock.

**Comment by lawand-dot-io** [comment]
The thing that I am curious about is that why does the window minimize animation not work? I feel like this might be related to this bug. Thanks for all the info so far, one more thing, which version of macOS are you on?

**Comment by lawand-dot-io** [comment]
And also do you use the AltTab app and if so, does the same bug happen when using it? Another thing, did you try restarting the system to see if the bug stops happening? It might just be a macOS quirk that is solved by a restart..

**Comment by hencomb** [comment]
Hi

It’s Sonoma 14.5

Cheers

**Comment by drjzhn** [comment]
Hey, I have the same bug. Sonoma 14.6.1. Have tried restarting and fresh install.
